scripts/save_logos.R

```diff
@@ -0,0 +1,32 @@
+
+# Script to move logos from this repository to the right repository
+# Select the svg logos
+svg_logos <- list.files("SVG", recursive = FALSE, full.names = TRUE)
+# list the folder with the package repositories
+path_to_dir_pkgs <- ".."
```

Comment:
this assumes you are in the folder that looks like

Rprojects/hex-sticker
Rprojects/package1
Rprojects/package2

something like that?

Reply:
Yes, this is how I have it on my local machine